roles/dnsmasq/tasks/main.yml

```diff
@@ -77,6 +77,6 @@
   wait_for:
     host: "{{dns_server}}"
     port: 53
-    delay: 5
-  when: inventory_hostname == groups['kube-node'][0]
+    timeout: 180
+  when: inventory_hostname == groups['kube-node'][0] and groups['kube-node'][0] in play_hosts
```

Comment:
ansible_play_hosts is the same. I'll use that one instead and update other refs to the deprecated var